app/src/data/ApiWrapper.ts

```diff
@@ -123,6 +125,12 @@ export class ApiWrapper {
         this.hostQuestionsResponse = new Fetcher<string>(
             `/api/v1/hostRegisterQuestions/${id}`
         )
+        this.hostShowstopperResponse = new Fetcher<string>(
```

Comment:
Please remove the "/api" prefix from these URLs, as that value is prepended by `Fetcher`'s constructor to potentially support other environments and API versions.